ftp-source/src/test/java/org/springframework/cloud/stream/module/ftp/FtpSourceIntegrationTests.java

```diff
@@ -103,6 +104,7 @@ public void create() throws IOException {
 		properties.put("remoteDir", ftpServer.getSourceFtpDirectory().getName());
 		properties.put("username", "foo");
 		properties.put("password", "foo");
+		properties.put("filenamePattern", "*");
 		properties.put("port", ftpServer.getPort());
 		properties.put("mode", "ref");
 		PropertiesInitializer.PROPERTIES = properties;
```

Comment:
How can you use that to inject dynamic properties, such as `ftpServer.getPort()` ?